service/src/main/java/bio/terra/workspace/service/resource/controlled/ControlledAiNotebookInstanceResource.java

```diff
@@ -24,6 +24,7 @@
 /** A {@link ControlledResource} for a Google AI Platform Notebook instance. */
 public class ControlledAiNotebookInstanceResource extends ControlledResource {
 
+  private static final String LOCATION = "us-central1-a";
```

Comment:
Do we need three location constants or should we share one?
Since this is an intermediate, it isn't too important.

Reply:
I created a resourceConstant file under controlled/